functions/__tests__/config.test.ts

```diff
@@ -77,7 +77,7 @@ describe('extensions config', () => {
   })
 
   // MEILISEARCH_INDEX_NAME
-  describe('Test meilisearchIndex parameters', () => {
+  describe('Test MeilisearchIndex parameters', () => {
```

Comment:
isn't `meilisearchIndex` a variable on your codebase?

Reply:
Nop I think we can even renamed it `Meilisearch Index`